cmd/kola/options.go

```diff
@@ -58,6 +63,7 @@ func init() {
 	sv(&kola.Options.BaseName, "basename", "kola", "Cluster name prefix")
 	ss("debug-systemd-unit", []string{}, "full-unit-name.service to enable SYSTEMD_LOG_LEVEL=debug on. Specify multiple times for multiple units.")
 	sv(&kola.UpdatePayloadFile, "update-payload", "", "Path to an update payload that should be made available to tests")
+	sv(&kola.Options.IgnitionVersion, "ignition-version", "", "Ignition version override: v2, v3")
```

Comment:
Just threw up `v2` & `v3` initially so we could bikeshed here.

Other ideas I tossed around were just using an int var and `s2/s3`

Reply:
I'm fine with either. Strings have the advantage of being able to be more fine grained in the future if we need it (not sure why we would though)